reference/functions/version_compare.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: version_compare
+published: true
+---
+
+[%CFEngine_function_prototype(string)%]
+
+**Description:** Returns `true` if the specified version comparison expression is true.
+
+[%CFEngine_function_attributes(string)%]
```

Comment:
```suggestion
[%CFEngine_function_attributes(version1, comparison, version2)%]
```
I think it's the same here, it may be that the parameter names need to match whatever is in the syntax description output for the function.

Reply:
Fixed, thank you :)